packages/shared/src/components/cards/v1/PostCardFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { CardImage, CardVideoImage } from './Card';
+import { Post, isVideoPost } from '../../../graphql/posts';
+import { cloudinary } from '../../../lib/image';
+
+interface PostCardFooterClassName {
+  image?: string;
+}
+
+type PostCardFooterProps = {
+  insaneMode: boolean;
+  openNewTab: boolean;
+  showImage: boolean;
+  post: Post;
+  className: PostCardFooterClassName;
+};
```

Comment:
To be inline with our guidelines, let's proceed with using interface for new ones.
```suggestion
interface PostCardFooterProps {
  insaneMode: boolean;
  openNewTab: boolean;
  showImage: boolean;
  post: Post;
  className: PostCardFooterClassName;
}
```

Reply:
Let's leave this for the implementation of the [card footer in WT-2007](https://dailydotdev.atlassian.net/browse/WT-2007).

Reply:
This is a new implementation, not sure we would have to wait the other ticket for this part.